reviewing-prs.md

```diff
@@ -91,9 +91,11 @@ This is the detailed code review of the actual changes, covering:
 
 Some code style guidelines can be found in the [Flink Code Style Page]({{ site.baseurl }}/contribute-code.html#code-style)
 
-## Review Template
+## Review with the @flinkbot
 
-The "Review Progress" section is posted at the end of the description of every new pull request.
+The Flink community is using a service called [@flinkbot](https://github.com/flinkbot) to help with the review of the pull requests.
```

Comment:
I guess this is a "chicken or the egg" problem, but the bot isn't active yet is it?